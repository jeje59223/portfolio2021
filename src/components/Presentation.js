import React from "react";
import moi from "../images/moi.jpeg";
import "../styles/presentation.css";

const Presentation = () => {
    return (
        <section id="presentation">
            <div className="container">
                <h2 className="text-center h2Presentation">Présentation</h2>
                <div className="row align-items-center">
                    <div className="col-md-6">
                        <h2>A Propos</h2>
                        <hr/>
                        <p>Bonjour, j'ai découvert le plaisir de coder en manipulant des cartes arduino et raspberry, 
                            j'y ai pris goût et ai décidé de me reconvertir vers un métier qui fait appel à ce type de 
                            compétence, c'est à dire développeur concepteur d'applications. J'ai beaucoup appris par 
                            moi-même via des cours sur le net. En 2019, j'ai fait un fongécif, chez M2I Formation Villeneuve d'Ascq 
                            pour découvrir le métier de développeur. J'ai eu une première expérience de développeur back-end PHP 
                            chez ITROOM sur Hem. Puis j'ai eu l'opportunité de participer à un bootcamp de trois mois chez Fewlines à 
                            Lille, qui m'a permis d'intégrer le siège Leroy Merlin en tant de développeur web.
                        </p>
                    </div>
                    <div className="col-md-6 text-center imagePresentation">
                        <img src={moi} alt="Jérôme Cnockaert"/>
                    </div>
                </div>
            </div>
        </section>
    )
}

export default Presentation;