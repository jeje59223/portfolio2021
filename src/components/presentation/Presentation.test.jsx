import toJson from 'enzyme-to-json';
import React from 'react';
import Adapter from 'enzyme-adapter-react-16';
import { shallow, configure } from 'enzyme';
import Presentation from "./Presentation";

configure({ adapter: new Adapter() });

describe('<Presentation>', () => {
  it('should render Presentation Component', () => {
    const wrapper = shallow(<Presentation />);
    const json = toJson(wrapper);
    expect(json).toMatchSnapshot();
  });
});