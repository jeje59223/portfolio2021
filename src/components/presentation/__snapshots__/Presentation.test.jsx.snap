// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Presentation> should render Presentation Component 1`] = `
<section
  id="presentation"
>
  <div
    className="container"
  >
    <h2
      className="text-center h2Presentation"
    >
      Présentation
    </h2>
    <div
      className="row align-items-center"
    >
      <div
        className="col-md-6"
      >
        <h2>
          A Propos
        </h2>
        <hr />
        <p>
          Bonjour, j'ai découvert le plaisir de coder en manipulant des cartes arduino et raspberry, j'y ai pris goût et ai décidé de me reconvertir vers un métier qui fait appel à ce type de compétence, c'est à dire développeur concepteur d'applications. J'ai beaucoup appris par moi-même via des cours sur le net. En 2019, j'ai fait un fongécif, chez M2I Formation Villeneuve d'Ascq pour découvrir le métier de développeur. J'ai eu une première expérience de développeur back-end PHP chez ITROOM sur Hem. Puis j'ai eu l'opportunité de participer à un bootcamp de trois mois chez Fewlines à Lille, qui m'a permis d'intégrer le siège Leroy Merlin en tant de développeur web.
        </p>
      </div>
      <div
        className="col-md-6 text-center imagePresentation"
      >
        <img
          alt="Jérôme Cnockaert"
          src="moi.jpeg"
        />
      </div>
    </div>
  </div>
</section>
`;
